code/API_definitions/geofencing.yaml

```diff
@@ -0,0 +1,658 @@
+        - webhook
+
+    SubscriptionInfo:
+      description: Represents a area-location event subscription.
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/CreateSubscription'
+        - type: object
+          properties:
+            eventSubscriptionId:
+              $ref: '#/components/schemas/SubscriptionId'
+            startsAt:
+              type: string
+              format: date-time
+              example: 2023-01-17T13:18:23.682Z
+              description: The time when the location-tracking started (e.g. in 7 days from now on). Provided in date-time format.
+            expiresAt:
+              type: string
+              format: date-time
+              example: 2023-01-17T13:18:23.682Z
+              description: The time when the location-tracking expired (e.g. in 7 days from now on). Provided in date-time format.
+          required:
+            - eventSubscriptionId
+            - webhook
+
+    EventTime:
+      format: date-time
+      type: string
+      description: |
+        Timestamp of when the occurrence happened. 
+        If the time of the occurrence cannot be determined then this attribute MAY be set to some other time (such as the current time) by the CloudEvents producer, 
+        however all producers for the same source MUST be consistent in this respect. In other words, 
+        either they all use the actual time of the occurrence or they all use the same algorithm to determine the value used. 
+        (must adhere on CAMARA datetime recommendation based on RFC 3339)
+
+    CloudEvent:
+      description: The Cloud-Event used for the callback.
+      type: object
+      required:
+        - id
+        - source
+        - type
+        - specversion
+        - time
+      properties:
+        id:
+          $ref: '#/components/schemas/EventId'
+        source:
+          description: |
+            identifies the context in which an event happened in the specific Provider Implementation. 
+            Often this will include information such as the type of the event source, the organization publishing the event or the process that produced the event. 
+            The exact syntax and semantics behind the data encoded in the URI is defined by the event producer.
+          type: string
+          example: "https://notificationSendServer12.supertelco.com"
+        type:
+          $ref: '#/components/schemas/Type'
+        specversion:
+          description: version of the specification to which this event conforms - must be "1.0"
+          type: string
+          example: "1.0"
+        datacontenttype:
+          description: media-type that describes the event payload encoding, must be application/json for CAMARA APIs
+          type: string
+          example: application/json
+        time:
+          $ref: '#/components/schemas/EventTime'
+        data:
+          $ref: '#/components/schemas/Data'
+      discriminator:
+        propertyName: 'type'
+        mapping:
```

Comment:
This is not correct, mapping must point to the schemas of the child objects. We must define separate objects per EventType. Look at examples in PRs for other APIs:
- DeviceStatus: https://github.com/camaraproject/DeviceStatus/pull/75
- QoD: https://github.com/camaraproject/QualityOnDemand/pull/224